rinad/src/ipcm/addons/ma/flowm.cc

```diff
@@ -357,7 +357,7 @@ void* ActiveWorker::run(void* param)
 			LOG_DBG("Connection stablished between MAD and Manager (port id: %u)", port_id);
 
 			//I/O loop
-			while(true) {
+			while(keep_running) {
```

Comment:
This sounds strange.

As I think I told you, the agreement with librina's readSDU API was that in case the flow was removed (due to IPCManager destruction => IPCP destruction => Flow destruction), librina should throw an exception.

Is this not happening any more? Is it broken?